lib/round-tracker.js

```diff
@@ -1,6 +1,8 @@
 import assert from 'node:assert'
 import * as Sentry from '@sentry/node'
-import { createMeridianContract } from './ie-contract.js'
+import { createMeridianContract, provider } from './ie-contract.js'
+import { onContractEvent } from 'on-contract-event'
+import { RPC_URL, GLIF_TOKEN } from '../spark-publish/ie-contract-config.js'
```

Comment:
I'd like to keep only one place where we are importing from spark-publish.

Would you mind re-exporting these constants in `./ie-contract.js`?

```suggestion
import { RPC_URL, GLIF_TOKEN, createMeridianContract, provider } from './ie-contract.js'
import { onContractEvent } from 'on-contract-event'
```